libs/coin-modules/coin-aptos/src/network/client.ts

```diff
@@ -236,4 +236,27 @@ export class AptosAPI {
       time: new Date(Number(block.block_timestamp)),
     };
   }
+
+  async getBalances(address: string): Promise<AptosBalance[]> {
+    if (!address) {
+      return [];
+    }
+
+    const response = await this.aptosClient.getCurrentFungibleAssetBalances({
+      options: {
+        offset: 0,
+        limit: 1000,
+        where: {
+          asset_type: { _eq: APTOS_ASSET_ID },
+          owner_address: { _eq: address },
+          token_standard: { _eq: "v1" },
```

Comment:
why setting to `token_standard`?

Reply:
I've tested and we don't neet to use the token_standard filter, I'll remove it